src/Backend/Snowflake/ToStage/FromABSCopyIntoAdapter.php

```diff
@@ -93,8 +94,8 @@ static function ($entry) use ($source) {
 CREDENTIALS=(AZURE_SAS_TOKEN=\'%s\')
 FILE_FORMAT = (TYPE=CSV %s)
 FILES = (%s)',
-            $this->connection->quoteIdentifier($destination->getSchemaName()),
```

Comment:
quoteSingleIdentifier by fungovalo normálně

Reply:
nefungovalo není to součást implementace toho SNFLK driveru pro dbal.

Reply:
Která platforma se používá?

Reply:
https://github.com/keboola/php-table-backend-utils/blob/master/src/Connection/Snowflake/SnowflakePlatform.php